stdlib/3.3/lzma.pyi

```diff
@@ -0,0 +1,105 @@
+import io
+import sys
+from typing import Any, BinaryIO, IO, List, Mapping, Optional, Union
+
+if sys.version_info >= (3, 6):
+    from os import PathLike
+    _PathOrFile = Union[str, bytes, IO[Any], PathLike[Any]]
+elif sys.version_info >= (3, 3):
+    _PathOrFile = Union[str, bytes, IO[Any]]
+else:
+    _PathOrFile = str
+
+_FilterChain = List[Mapping[str, Any]]
+
+FORMAT_AUTO: int
+FORMAT_XZ: int
+FORMAT_ALONE: int
+FORMAT_RAW: int
+CHECK_NONE: int
+CHECK_CRC32: int
+CHECK_CRC64: int
+CHECK_SHA256: int
+CHECK_ID_MAX: int
+CHECK_UNKNOWN: int
+FILTER_LZMA1: int
+FILTER_LZMA2: int
+FILTER_DELTA: int
+FILTER_X86: int
+FILTER_IA64: int
+FILTER_ARM: int
+FILTER_ARMTHUMB: int
+FILTER_SPARC: int
+FILTER_POWERPC: int
+MF_HC3: int
+MF_HC4: int
+MF_BT2: int
+MF_BT3: int
+MF_BT4: int
+MODE_FAST: int
+MODE_NORMAL: int
+PRESET_DEFAULT: int
+PRESET_EXTREME: int
+
```

Comment:
You're missing `is_check_supported` from https://docs.python.org/3/library/lzma.html#miscellaneous, and the exception class `LZMAError`.

Reply:
oops, 444d0ccb1dee0129fe18da07ea6dd5e8f0e1410b and cb0e8acbae76aefef8a9dc792563c82ba5463410 add those.